assets/components/src/with-wizard/index.js

```diff
@@ -145,34 +146,47 @@ export default function withWizard( WrappedComponent, requiredPlugins ) {
 		/**
 		 * Begin loading.
 		 */
-		startLoading = () => {
-			this.setState( state => ( {
-				loading: state.loading + 1,
-			} ) );
+		startLoading = quiet => {
+			if ( quiet ) {
+				this.setState( state => ( {
+					quietLoading: state.quietLoading + 1,
```

Comment:
I know this is true in `master` too, but what's the reasoning behind the type coercion and `0`/`1` instead of just using `true` and `false` for these state values? Seems to me like just sticking with booleans would be a bit more straightforward and not functionally different.

Reply:
I believe the idea is that there can be multiple concurrent API requests in flight simultaneously. By incrementing and decrementing a counter, the loading state will only be removed once all API requests finish up.